RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -648,6 +709,79 @@ extension StrokeStyle {
 	static let transactionReview = StrokeStyle(lineWidth: 2, dash: [5, 5])
 }
 
+extension TransactionReview {
+	public typealias ValidatorsState = ValidatorsView.ViewState
+	public typealias ValidatorState = ValidatorView.ViewState
+
+	public struct ValidatorsView: SwiftUI.View {
```

Comment:
Should this be here? can it be moved to a separate file?

Reply:
It definitely could, but it's only used here I think, and lots of other subviews are in this file.